cmd/ui/src/views/Content.tsx

```diff
@@ -50,6 +50,7 @@ const Content: React.FC = () => {
         if (isFullyAuthenticated) {
             dispatch(fetchAssetGroups());
             dispatch(ListAssetGroups());
+            dispatch(fetchCustomNodeInformation());
```

Comment:
running this at this level in the component tree would result on this request occurring on pages that dont require custom nodes. Do we need to run it at this level, or is there a more focused/narrow component that would work?